.github/actions/mirror/action.yml

```diff
@@ -0,0 +1,20 @@
+name: 'mirror'
+inputs:
+  from:
+    default: "camunda/zeebe:8.2.7"
```

Comment:
What is the idea of this default? If it's an example, then maybe we shouldn't set default but provide it as part of the description.

Reply:
The idea is when  you mirror an image by filling out this form:

![2024-10-21-153940_grim](https://github.com/user-attachments/assets/3391b054-ad8e-4667-87f9-062c0d2b016e)

you will have a valid entry pre-selected and you as a user only needs to modify the part of the form that you want to change. And my thought for the future of this would be  if renovate updated that default value.

But some parts of that doesn't make sense. The `action.yml` for the mirroring action does not have a form like the other workflow_dispatch github actions.

Reply:
I removed the default and added it as part of the description.